hw/ip/adc_ctrl/data/adc_ctrl.hjson

```diff
@@ -54,6 +54,11 @@
       local:   "true",
     }
   ],
+  countermeasures: [
+    { name: "BUS.INTEGRITY",
+      desc: "End-to-end bus integrity scheme."
```

Comment:
this is not follow the example in
https://docs.google.com/presentation/d/1F8gk-WcxwF-g2mn29A_ko0De8amiyv965Od3FKJCj0U/edit#slide=id.gfa6dc7b147_0_7

I used that format in #9572, which I think I like better.